### DISCUSSION TITLE
wormhole wETH on Terra

### DISCUSSION THREAD

**Discussion by xiong69** [wormhole-foundation/wormhole#555] [Developer Q&A] [answered]
Hello

I transfer ETH Ethereum to Terra and i got **wormhole wETH**. I can't do anything with it and want to return this liquidity to anything usefully.
Thanks

**Comment by justinschuldt** [comment, marked as answer]
You can simply transfer it back to Ethereum.

**Reply by phawat4256** [reply]
how to back weth,pls.
